mediacenter/views.py

```diff
@@ -0,0 +1,38 @@
+from django.conf import settings
+from django.contrib.admin.views.decorators import staff_member_required
+from django.core.urlresolvers import reverse_lazy
+from django.views.generic import (CreateView, DeleteView, DetailView, ListView,
+                                  UpdateView)
+
+from .models import Document
+
+
+class Index(ListView):
+    model = Document
+    template_name = 'mediacenter/index.html'
```

Comment:
No really strong argument actually, I just thought this page was more about a section home page than a list of objects, so that the template will ends with more elements than just a list of object we would expect from a "document_list.html" template.